lib/tasks/docs.rake

```diff
@@ -0,0 +1,14 @@
+require 'rdoc/task'
+
+namespace :api_docs do
+  RDoc::Task.new :generate do |rdoc|
+    rdoc.main = "API_README.md"
+    rdoc.rdoc_files.include(
+      "app/controllers/notifications_controller.rb",
+      "app/controllers/users_controller.rb",
+      "API_README.md",
+    )
```

Comment:
Maybe splat this for the controller files at least? Or would the output be funny looking without any docs if they aren't present?

Reply:
I'm not sure what you mean, tbh

Reply:
Sorry I must have been tired when I wrote... I meant [glob](https://ruby-doc.org/core-2.4.2/Dir.html#method-c-glob). `app/controllers/*_controller.rb` instead of forgetting to add a line when we make a new controller.

Reply:
Oh, I chose specific ones because the other controllers actually _dont_ have api endpoints

Reply:
Oooh I just thought they'd be ignored if that's the case. Well, nevermind then! 👍